### PR TITLE
[Env] Upgrade php version from 8.1 to 8.2

### LINKED ISSUE #257: [1.x] Upgrade to PHP 8.2
### Description

Upgrade to PHP 8.2

### Example

_No response_

### PR DESCRIPTION
**Pull request by Moemen-Gaballah**
| Q             | A                                                                  |
|---------------|--------------------------------------------------------------------|
| Branch?       | 1.x                                              |
| Bug fix?      | no                                                             |
| New feature?  | no                   |
| Deprecations? | no  |
| Tickets       | Fix #257             |
| License       | MIT                                                                |

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -81,8 +81,8 @@
     "symfony/polyfill-php81": "*"
   },
   "scripts": {
-    "check-style": "vendor/bin/php-cs-fixer fix --dry-run --allow-risky=yes --format=txt --verbose",
-    "format": "vendor/bin/php-cs-fixer fix --allow-risky=yes",
+    "check-style": "PHP_CS_FIXER_IGNORE_ENV=1 vendor/bin/php-cs-fixer fix --dry-run --allow-risky=yes --format=txt --verbose",
```
Comment (imdhemy) [inline composer.json:84]: I don't see this a good practice. This creates a technical debt. 

Please, change all configs related the PHP version and upgrade the packages.

**Review by imdhemy — CHANGES_REQUESTED** [review]
@Moemen-Gaballah Could you please address the comment I left?

File: composer.json (hunk 1 above)
Comment (Moemen-Gaballah) [reply composer.json:84]: @imdhemy 
upgrade all packages or special friendsofphp/php-cs-fixer?
friendsofphp/php-cs-fixer is current version latest

Comment (imdhemy) [reply composer.json]: All of them

**Pull request closed without merging**